integreat_cms/static/src/js/analytics/hix-widget.ts

```diff
@@ -76,25 +76,30 @@ const updateHixStateLabel = (state: string) => {
  * Display the HIX feedback details
  */
 const updateHixFeedback = (hixFeedback: string) => {
-    const feedbackSections = document.querySelectorAll("[hix-feedback-category]");
-    const feedbackJson = JSON.parse(hixFeedback);
-
     let feedbackCount = 0;
+    if (hixFeedback !== "None") {
+        const feedbackSections = document.querySelectorAll("[hix-feedback-category]");
+        const feedbackJson = JSON.parse(hixFeedback);
```

Comment:
`None ` no longer triggers the empty "for review" part, but leaving it completely empty does (and so do some other non-sense values, I presume).

IMO it's probably fine, but a more robust solution might be attempting to parse the JSON, and in case that fails, to then not go in the inner loop?

Reply:
That sounds nice 😸 Applied!